tests/test_repo_interactions.py

```diff
@@ -239,7 +237,7 @@ def test_chartpress_paths_configuration(git_repo, capfd):
     open("extra-chart-path.txt", "w").close()
     git_repo.git.add(all=True)
     sha = git_repo.index.commit("Added extra-chart-path.txt").hexsha[:7]
-    tag = f"0.0.1-{PRERELEASE_PREFIX}3.h{sha}"
+    tag = f"0.0.1-0.dev.git.3.h{sha}"
```

Comment:
I see `{PRERELEASE_PREFIX}` be used sometimes, but not other times, that doesn't seem intentional to me - is it?

Reply:
I kept changing my mind whether it was worth having as a constant, which is why I haven't been super consistent. I'm still not sure!

In the latest iteration, there are also two prefixes, so what should the variable be is ambiguous:

- `git.` - the prefix we _always_ use ahead of the `.{n}.h{sha}`
- `0.dev.git.` - the full prefix we use if there was no prerelease tag already

Reply:
Also ambiguous: should the constant include the trailing `.`? I think it reads better if it doesn't, but then can't be empty.

Reply:
Went with `GIT_PREFIX="git"` and `PRERELEASE_PREFIX="0.dev.git"`, so excluding trailing `.`.